diff --git a/node_modules/react-native-vector-icons/android/build.gradle b/node_modules/react-native-vector-icons/android/build.gradle
index 28e2fcc..82c7972 100644
--- a/node_modules/react-native-vector-icons/android/build.gradle
+++ b/node_modules/react-native-vector-icons/android/build.gradle
@@ -1,34 +1,30 @@
-buildscript {
-  repositories {
-    jcenter()
-  }
+apply plugin: "com.android.library"
 
-  dependencies {
-    classpath 'com.android.tools.build:gradle:2.3.+'
-  }
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
 }
 
-apply plugin: 'com.android.library'
-
 android {
-  compileSdkVersion 26
-  buildToolsVersion "26.0.1"
+    namespace "com.oblador.vectoricons"
+    compileSdk safeExtGet("compileSdkVersion", 36)
+
+    defaultConfig {
+        minSdkVersion safeExtGet("minSdkVersion", 24)
+        targetSdkVersion safeExtGet("targetSdkVersion", 36)
+        versionCode 1
+        versionName "1.0"
+    }
 
-  defaultConfig {
-    minSdkVersion 16
-    targetSdkVersion 26
-    versionCode 1
-    versionName "1.0"
-  }
-  lintOptions {
-    abortOnError false
-  }
+    lintOptions {
+        abortOnError false
+    }
 }
 
 repositories {
-  mavenCentral()
+    google()
+    mavenCentral()
 }
 
 dependencies {
-  compile "com.facebook.react:react-native:+"
+    implementation "com.facebook.react:react-native:+"
 }
